htdocs/content/crypto.php

```diff
@@ -193,75 +193,85 @@
 										// Do the scan if chance >= 0
 										if ($chance >= 0)
```

Comment:
Do we need to adjust the calculation of `$chance` as well? It has the same `mt_rand(0,2)-1` calculation in there

Reply:
I'm not sure if that's necessary. At least for 3.6. For me, success chance was always ok and better balanced than the overall results after passing this threshold.